lib/limits.c

```diff
@@ -60,19 +62,15 @@ static int setrlimit_value (unsigned int resource,
 		 * Also, we are limited to base 10 here (hex numbers will not
 		 * work with the limit string parser as is anyway)
 		 */
-		char *endptr;
-		long longlimit = strtol (value, &endptr, 10);
+		l = strtol(value, &endptr, 10);
```

Comment:
I don't this variable name change from longlimit to l. Can you propose a more meaningful name?

Reply:
There are two reasons to rename it from `longlimit` to `l`.

The first one is that `longlimit` is a very long name that made me split a long line (think of `if (__builtin_mul_overflow(longlimit, multiplier, &limit)) {`.

The second (more subjective) is that it is used as a temporary variable for holding the `limit` before actually storing it in `limit`.  Since the variable is only used in 2 lines (the first one where we write to it, and the second one where we read it), it is quite obvious what it is for.

Google's C++ style guide says
> Generally speaking, descriptiveness should be proportional to the name's scope of visibility. For example, n may be a fine name within a 5-line function, but within the scope of a class, it's likely too vague.
<https://google.github.io/styleguide/cppguide.html#General_Naming_Rules>

I find `l` appropriate here, for a variable that is used in 2 lines, and is a temporary for `limit`.

Reply:
I don't have any problem to changing its name as long as it is something meaningful. Take into account that you also increased its scope, so now we have a variable named `l` that could be used in the 40 lines of the function.

This is just an opinion so I will not insist.